onelogin_aws_cli/configuration.py

```diff
@@ -22,20 +22,35 @@ def __init__(self, config_file=None):
         if self.file is not None:
             self.load()
 
+    @property
+    def has_defaults(self) -> bool:
+        """True if the default section is non-empty"""
+        return len(self.items(self.default_section)) > 0
+
     @property
     def is_initialised(self) -> bool:
         """True if there is at least one section"""
-        return (len(self.sections()) > 0)
+        return (len(self.sections()) > 0) or self.has_defaults
 
     def load(self):
         self.read_file(self.file)
+        # For backwards compatibility, we check if they have a default
+        # section instead of a defaults section and do a little switcheroo as
+        # needed.
+        if self.has_section('default') and not self.has_defaults:
+            print("It looks like you're using a the deprecated 'default' "
+                  "section.\nConsider renaming the section to 'defaults'.")
+            self.default_section = 'default'
 
     def initialise(self, config_name='defaults'):
         """
         Prompt the user for configurations, and save them to the
         onelogin-aws-cli config file
         """
         print("Configure Onelogin and AWS\n\n")
+        if not config_name:
```

Comment:
This will never be falsey unless calling functions pass a `None` value in. If that's the case, the default value for the method param should probably just be `config_name = None`.

Reply:
Oops, this got leftover from the merge.